app/controllers/editLiability/DisposeLiabilitySummaryController.scala

```diff
@@ -80,7 +80,7 @@ class DisposeLiabilitySummaryController @Inject()(mcc: MessagesControllerCompone
         }
           yield {
             retrieveLiabilityReturn match {
-              case Some(x) => Ok(views.html.editLiability.disposeLiabilityPrintFriendly(x, organisationName))
+              case Some(_) => Ok(views.html.editLiability.disposeLiabilityPrintFriendly(retrieveLiabilityReturn.get, organisationName))
```

Comment:
This was best as it was really because on the left side the Option had already been opened up to reveal the contents. You could rename "x" to be more descriptive instead of you want to improve it?

Reply:
Thought about changing this to liabilityData but as x is used elsewhere in the controller decided to revert back to the original and retested